src/BayesFilters/include/BayesFilters/sigma_point.h

```diff
@@ -12,12 +18,39 @@ namespace bfl
 {
 namespace sigma_point
 {
+    using FunctionEvaluation = std::function<std::pair<bool, bfl::Data>(const Eigen::Ref<const Eigen::MatrixXf>&)>;
+    
+    struct UTWeight
+    {
+        Eigen::VectorXd mean;
+        Eigen::VectorXd cov;
+        /**
+         * c = sqrt(n + lambda) with lambda a ut parameter.
+         */
+        double c; 
 
-    void unscented_weights(const unsigned int n, const double alpha, const double beta, const double kappa,
-                           Eigen::Ref<Eigen::VectorXd> weight_mean, Eigen::Ref<Eigen::VectorXd> weight_covariance, double& c);
+        UTWeight(size_t n, const double alpha, const double beta, const double kappa);
```

Comment:
Please use `std::size_t` instead of `size_t`. Check it also in the other commits 👍

Reply:
Sure! :+1: